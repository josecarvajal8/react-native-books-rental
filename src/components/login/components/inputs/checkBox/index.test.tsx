import React from 'react';
import renderer from 'react-test-renderer';
import CheckBox from '.';
const mockProps = {
    onChangeValue: () => null,
    fieldId: 'test',
    label: 'test_field',
    value: false
}
test('Render checkBox component correctly', () => {
    const checkBoxCmp = renderer.create(<CheckBox {...mockProps} />).toJSON();
    expect(checkBoxCmp).toMatchSnapshot();
});