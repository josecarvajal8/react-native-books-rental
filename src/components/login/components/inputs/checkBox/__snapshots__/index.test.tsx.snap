// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render checkBox component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
  >
    <View
      accessible={true}
      style={
        Object {
          "alignItems": "center",
          "borderWidth": 1,
          "height": 18,
          "justifyContent": "center",
          "marginRight": 10,
          "opacity": 1,
          "width": 18,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "height": 12,
            "width": 12,
          }
        }
      />
    </View>
  </RNGestureHandlerButton>
  <Text
    style={
      Object {
        "fontFamily": "Montserrat-Regular",
      }
    }
  >
    [missing "en.test_field" translation]
  </Text>
</View>
`;
